src/content/docs/accounts/accounts-billing/account-structure/add-accounts.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Add accounts
```

Comment:
This new doc feels weirdly short. But... I think that's probably all right?

Reply:
i think it's okay. adding accounts is a pretty huge ask and source of issues so I wanted to make it more visible in sidebar and that's where I'd expect to find it.